src/TemporaryUploadedFile.php

```diff
@@ -142,20 +142,51 @@ public static function createFromLivewire($filePath)
 
     public static function canUnserialize($subject)
     {
-        if (! is_string($subject)) return false;
+        if (is_string($subject)) {
+            return Str::startsWith($subject, 'livewire-file:')
+                || Str::startsWith($subject, 'livewire-files:');
+        } elseif (is_array($subject)) {
```

Comment:
I would break this elseif out into a standalone "if". It will make it look nicer and less like a big nested chunk of code.
The first if returns so this is possible

Reply:
updated my pr, thanks @calebporzio